speed.c

```diff
@@ -59,8 +58,7 @@ static float _calculate_search_linear_speed(float force)
  */
 void set_linear_speed_variables(float force, bool run)
 {
-	linear_acceleration = 2 * force / MOUSE_MASS;
-	linear_deceleration = 2 * force / MOUSE_MASS;
+	max_force = force;
```

Comment:
- I would take out the force set, and keep just max_linear_speed. The function name would be then 'set_max_linear_speed'. To update max_force I would always use 'set_max_force' as for calibration or move files. 'set_linear_speed_variables' is only used on main, so there I would add also set_max_force.

Reply:
- If you do not agree and want to keep the function as it is, change the docstring to add max_force.

Reply:
I changed the name of the function and its description instead. How do you like that? :blush:

Reply:
perfect :+1: